nlp/notebooks/stac_wrapper/stac_config.cfg

```diff
@@ -0,0 +1,20 @@
+[PEPS]
+PEPS_url = https://peps.cnes.fr/resto/api/collections/
+# PEPS_collections = ["S1", "S2", "S2ST", "S3"]
+# PEPS_formats = ["json", "atom"]
+
+[COPERNICUS]
+COP_url = https://scihub.copernicus.eu/dhus/search?q=
+COP_user = None
+COP_pwd = None
+
+[PAVICCS]
+PAVICCS_url = https://pavics.ouranos.ca/twitcher/ows/proxy/catalog/wps
```

Comment:
This seems to point at a catalog that is no longer maintained. 
How was it used?

Reply:
in nlp/notebooks/stac_wrapper/DACCS_connecteurs_externes.ipynb
 wps = WPSClient(config['PAVICS_url'])
resp = wps.pavicsearch(constraints=params, limit=10, type="File")

Reply:
Although it is still referenced in the notebook, the "catalog/wps" endpoint is not available anymore on PAVICS.
This is not a STAC catalog either, but a WPS service, so it doesn't make much sence in a STAC config.
For now PAVICS does not have STAC enabled, but if they enable it, it should be by default under https://pavics.ouranos.ca/stac

I also noticed following changes : 
- PEPS now at: https://peps.cnes.fr/resto/collections/ (no `/api` before `/collections`)
  example: https://peps.cnes.fr/resto/collections/S1/1e09aa1f-140d-58cd-8f55-8db8fdb60edd.json
- Copernicus' STAC now seems to be at: https://catalogue.dataspace.copernicus.eu/stac

Reply:
wow, you really went deep into it. do you want me to change the config to the new urls for PEPS and Copernicus or you can do it?
As for Pavics, I would leave the code as it is, as it was done in 2019, and even if it's not available anymore, it shows our work done on the project.

Reply:
no, it doesn't. it was a separate exploration of external connectors